indigo/distribution.yaml

```diff
@@ -3320,14 +3320,16 @@ repositories:
       - grid_map_filters
       - grid_map_loader
       - grid_map_msgs
+      - grid_map_pcl
       - grid_map_ros
       - grid_map_rviz_plugin
       - grid_map_visualization
       tags:
         release: release/indigo/{package}/{version}
       url: https://github.com/ethz-asl/grid_map-release.git
-      version: 1.4.0-0
+      version: 1.4.1-0
     source:
+      test_pull_requests: true
```

Comment:
Since this repo is not in the `ros` org unit the pull request integration will only work if you either give access to the @ros-pull-request-builder user or setup the webhooks manually: see http://wiki.ros.org/buildfarm/Pull%20request%20testing